server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -0,0 +1,36 @@
+package views.admin.programs;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static views.BaseHtmlView.renderHeader;
+
+import com.google.inject.Inject;
+import j2html.tags.specialized.H1Tag;
+import play.mvc.Http;
+import play.twirl.api.Content;
+import services.program.ProgramDefinition;
+import views.HtmlBundle;
+import views.admin.AdminLayout;
+import views.admin.AdminLayoutFactory;
+
+/** A view for admins to update the image associated with a particular program. */
+public final class ProgramImageView {
+  private final AdminLayout layout;
+
+  @Inject
+  public ProgramImageView(AdminLayoutFactory layoutFactory) {
+    this.layout = checkNotNull(layoutFactory).getLayout(AdminLayout.NavPage.PROGRAMS);
+  }
+
+  /**
+   * Renders the image currently associated with the program and a form to add / edit / delete the
```

Comment:
suggestion (non-blocking): Add a todo since this does not currently do what it says in the comment.

Reply:
Done!